spec/requests/admin/supplier_users_spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Admin::SupplierUsers", type: :request do
+  let(:lead_provider) { create(:lead_provider) }
+  let(:delivery_partner) { create(:delivery_partner) }
+  let(:full_name) { Faker::Name.name }
+  let(:email) { Faker::Internet.email }
+
+  before do
+    admin_user = create(:user, :admin)
+    sign_in admin_user
+  end
+
+  describe "GET /admin/suppliers/users" do
+    it "renders the correct template" do
```

Comment:
`correct template` should probably be specifying which template exactly.

`renders the index template`?